pkg/app/pipedv1/plugin/inputs.go

```diff
@@ -0,0 +1,24 @@
+package plugin
```

Comment:
Please add the file header copyright

Reply:
Thank you.
I added it with this commit
https://github.com/pipe-cd/pipecd/pull/5020/commits/0b562e4cb0aed3d16f412c0e2222f42a7df566ff